src/strings/PowerFxResources.en-US.resx

```diff
@@ -4623,6 +4627,16 @@
       A partial operator is the 2nd part of a statement `[Partial Op]` and can be one of "And", "Or", "Table" or "Record".
       It's used to determine how to combine multiple expressions with the same name and operator.</comment>
   </data>
+  <data name="AboutCollect_data_source" xml:space="preserve">
+    <value>The data source that you want to add data to.</value>
```

Comment:
not sure if we want "you" in this description. Maybe "The datasource to add data to"